src/libsyntax/parse/parser.rs

```diff
@@ -6898,12 +6902,14 @@ impl<'a> Parser<'a> {
         }
     }
 
-    fn parse_record_struct_body(&mut self) -> PResult<'a, Vec<StructField>> {
+    fn parse_record_struct_body(&mut self) -> PResult<'a, (Vec<StructField>, bool)> {
```

Comment:
```suggestion
    fn parse_record_struct_body(&mut self) -> PResult<'a, (Vec<StructField>, /*recovered*/ bool)> {
```
:slightly_smiling_face: